bogenliga/bogenliga-application/src/main/java/de/bogenliga/application/springconfiguration/security/jsonwebtoken/JwtTokenProvider.java

```diff
@@ -180,13 +180,23 @@ private String createToken(final String username, final Long id, final Long vers
         final Date now = new Date();
         final Date validity = new Date(now.getTime() + validityInMilliseconds);
 
+        // if user is default user (not logged in user) expiration time is not set
+        // since the default should be logged in "forever"
+        if(username.equals("ligadefault")) {
```

Comment:
den String "ligadefault" bitte in eine Klassenkonstante auslagern als private static final String DEFAULT_USER_NAME = "ligadefault"